code/parachain/frame/staking-rewards/src/test/test_reward_accumulation_hook.rs

```diff
@@ -244,11 +224,8 @@ fn test_accumulate_rewards_pool_empty_refill() {
 
 		progress_to_block(current_block + 1, &mut current_block);
 
-		check_events([crate::Event::<Test>::RewardAccumulationHookError {
-			pool_id: A::ID,
-			asset_id: A::ID,
-			error: RewardAccumulationHookError::RewardsPotEmpty,
-		}]);
+		// Note: reward pot for A_B should become empty, once it adds the remaining rewards.
+		// Should we emit some event on the pallet? If yes, it should be asserted here.
```

Comment:
an event is emitted, see `reward_accumulation_hook_reward_update_calculation`. We could consider a new event for this though; thoughts @vimukthi-git?

Something like `RewardsPoolPaused` and `RewardPoolUnpaused`?

Reply:
Yes sorry, that's what I meant. A specific event for when we pause/unpause. I think we should.

Reply:
I think that would be a good change, but I'm OK if we get this merged first and then update the event(s) in a followup PR so this one doesn't get too big

Reply:
Sure let's do that. I created a task for this, which I'll do once this PR is merged: https://app.clickup.com/t/3qftw8g